### PR TITLE
[ID] Update `Client` translation, create `Release stream` and `How to search for and download beatmaps ` (placeholder) translations.

### PR DESCRIPTION
**Pull request by FazlyMR**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- TicClick: `SKIP_WIKILINK_CHECK` (`How to search for and download beatmaps` is one of the exceptions for now)

### CODE REVIEW

File: wiki/Client/Release_stream/id.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+stub: true
+---
+
+# Versi rilis
+
+osu! memiliki beberapa **versi rilis**, atau versi klien permainan, yang mendapatkan pembaruan dengan jumlah yang berbeda-beda. Agar dapat menangani masalah-masalah secara dini, fitur dan perubahan-perubahan baru akan dicoba di dalam versi-versi *preview* (pratinjau) sebelum akhirnya ditambahkan ke dalam versi stable.
```
Comment (wowcake) [inline wiki/Client/Release_stream/id.md:7]:
```suggestion
osu! memiliki beberapa **versi rilis**, atau versi klien permainan, yang mendapatkan pembaruan dengan jumlah yang berbeda-beda. Agar dapat menangani masalah-masalah secara dini, fitur dan perubahan-perubahan baru akan ditambahkan terlebih dahulu di dalam versi-versi *preview* (pratinjau) untuk dicoba sebelum akhirnya ditambahkan ke dalam versi stable.
```

**Review by wowcake — CHANGES_REQUESTED** [review]

File: wiki/Client/Release_stream/id.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+# Versi rilis
+
+osu! memiliki beberapa **versi rilis**, atau versi klien permainan, yang mendapatkan pembaruan dengan jumlah yang berbeda-beda. Agar dapat menangani masalah-masalah secara dini, fitur dan perubahan-perubahan baru akan ditambahkan terlebih dahulu di dalam versi-versi *preview* (pratinjau) untuk dicoba sebelum akhirnya ditambahkan ke dalam versi stable.
+
+| Versi rilis | Catatan |
+| :-- | :-- |
+| Stable | Terkini, versi paling populer; tidak mendapat fitur baru; menerima pembaruan setelah Beta |
+| Beta | Menerima pembaruan setelah Cutting Edge; tidak mendapat fitur baru |
```
Comment (Nivalyx) [inline wiki/Client/Release_stream/id.md:12]:
```suggestion
| Beta | Menerima pembaruan setelah Cutting Edge; tidak lagi memperoleh fitur baru |
```

File: wiki/Client/Release_stream/id.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+| Cutting Edge | Mendapat pembaruan terbaru; tidak mendapat fitur baru |
```
Comment (Nivalyx) [inline wiki/Client/Release_stream/id.md:13]:
```suggestion
| Cutting Edge | Menerima pembaruan paling pertama; tidak lagi memperoleh fitur baru |
```

File: wiki/Client/Release_stream/id.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+stub: true
+---
+
+# Versi rilis
+
+osu! memiliki beberapa **versi rilis**, atau versi klien permainan, yang mendapatkan pembaruan dengan jumlah yang berbeda-beda. Agar dapat menangani masalah-masalah secara dini, fitur dan perubahan-perubahan baru akan ditambahkan terlebih dahulu di dalam versi-versi *preview* (pratinjau) untuk dicoba sebelum akhirnya ditambahkan ke dalam versi stable.
```
[the remark this answers is not in the document]
Comment (FazlyMR) [reply wiki/Client/Release_stream/id.md:7]: This one is nicer to read for sure.

**Review by Nivalyx — APPROVED** [review]

**Pull request merged**